### PR TITLE
Fix header documentation

### PR DESCRIPTION
**Pull request by teetak01**
Moved tr_info to right place in header documentation

### CODE REVIEW

**Comment by teetak01** [comment]
@jupe @tommikas

File: README.md [hunk 1]
```diff
@@ -150,9 +150,9 @@ int main(void){
     mbed_trace_mutex_release_function_set( my_mutex_release ); // only if thread safety is needed
     mbed_trace_init();       // initialize the trace library
     tr_debug("this is debug msg");  //-> "[DBG ][main]: this is a debug msg"
+    tr_info("this is info msg");    //-> "[INFO][main]: this is an info msg"
     tr_err("this is error msg");    //-> "[ERR ][main]: this is an error msg"
     tr_warn("this is warning msg"); //-> "[WARN][main]: this is a warning msg"
```
Comment (tommikas) [inline README.md]: These two above lines should also be switched around.

File: mbed-trace/mbed_trace.h [hunk 2]
```diff
@@ -30,9 +30,9 @@
  *      int main(void){
  *          mbed_trace_init();   // initialize trace library
  *          tr_debug("this is debug msg");  //print debug message to stdout: "[DBG]
+ *          tr_info("this is info msg");
  *          tr_err("this is error msg");
  *          tr_warn("this is warning msg");
```
Comment (tommikas) [inline mbed-trace/mbed_trace.h]: Here too.

**Comment by teetak01** [comment]
Fixed

**Comment by teetak01** [comment]
Is 5 commits enough to fix documentation?

**Comment by tommikas** [comment]
I think that should finally be it. At least didn't notice any other instances of this in the two files...

**Pull request merged**